server/controllers/order.js

```diff
@@ -0,0 +1,132 @@
+/* eslint no-shadow: "off" */
+/* eslint max-len: "off" */
+/* eslint arrow-body-style: "off" */
+/* eslint linebreak-style: "off" */
+/* eslint object-curly-newline: "off" */
+/* eslint no-template-curly-in-string: "off" */
+/* eslint no-param-reassign: "off" */
+
+import db from '../db';
+
+/** food order controller class */
+class OrderController {
+  /**
+* @function addToCart
+* @memberof OrderController
+*
+* @param {Object} req - this is a request object that contains whatever is requested for
+* @param {Object} res - this is a response object to be sent after attending to a request
+*
+* @static
+*/
+
+  static addToCart(req, res) {
+    const { userId } = req;
+    let { foodId, quantity } = req.body;
+    foodId = foodId ? foodId.toString().replace(/\s+/g, '') : foodId;
+    quantity = quantity ? quantity.toString().replace(/\s+/g, '') : quantity;
+
+    db.task('post meal', data => data.menu.findById(foodId)
+      .then((meal) => {
+        if (!meal) {
+          return res.status(404).json({
+            success: 'false',
+            message: 'meal not found',
+          });
+        }
+        const total = quantity * meal.price;
+        const itemName = meal.item_name;
+        const mealItem = {
+          userId,
+          foodId,
+          itemName,
+          quantity,
+          total,
+        };
+        return db.cart.create({ userId, foodId, itemName, quantity, total })
```

Comment:
what is the difference between foodId and itemName? Is itemName not the same as foodName